var/spack/repos/builtin/packages/libdrm/package.py

```diff
@@ -27,12 +27,24 @@ class Libdrm(Package):
     version('2.4.59',  sha256='ed9d03a92c2d80e6310cc350db3430620f1659ae084a07c6824cee7bc81ae8fa')
     version('2.4.33',  sha256='bd2a8fecf28616f2157ca33ede691c139cc294ed2d0c4244b62ca7d22e98e5a4')
 
+    variant('docs', default=False, description="Build man pages")
+
     depends_on('pkgconfig', type='build')
-    depends_on('docbook-xml', type='build')
-    depends_on('docbook-xsl', type='build')
     depends_on('libpciaccess@0.10:')
     depends_on('libpthread-stubs')
-    depends_on('meson', type='build', when='@2.4.101:')
+
+    # 2.4.90 is the first version to use meson.
+    depends_on('meson', type='build', when='@2.4.90:')
+
```

Comment:
While @2.4.90: supports meson, spack has supported libdrm as an autotools-only package through 2.4.100 (see https://github.com/spack/spack/pull/27253). I think that retroactively making meson the default option starting with 2.4.90 may result in users with e.g. previously concretized environments that have always used the autotools version suddenly finding themselves with the meson version. For this reason, I introduced meson as the build system only with new versions after 2.4.101: when the autotools option was removed entirely. I think we may want to maintain that distinction in the interest of not changing the behavior of existing packages/versions.

Reply:
Okay, I doubt people use 2.4.100 exactly, but I can see the point.